packages/ti_eclecticiq/data_stream/outgoing_feed/fields/ecs.yml

```
@@ -0,0 +1,94 @@
+- external: ecs
+  name: ecs.version
+- external: ecs
+  name: message
+- external: ecs
+  name: tags
+- external: ecs
+  name: event.url
+- external: ecs
+  name: event.created
+- external: ecs
+  name: event.original
+- external: ecs
+  name: event.start
+- external: ecs
+  name: event.end
+- external: ecs
+  name: event.provider
+- external: ecs
+  name: event.category
+- external: ecs
+  name: event.kind
+- external: ecs
+  name: event.type
+- external: ecs
+  name: threat.indicator.first_seen
+- external: ecs
+  name: threat.indicator.confidence
+- external: ecs
+  name: threat.indicator.marking.tlp
+- external: ecs
+  name: threat.indicator.name
```

```suggestion
- name: threat.indicator.name
  level: extended
  type: keyword
  description: The display name indicator in an UI friendly format
```

Since the fix is not available in `8.9.2`, above workaround is suggested.

---

@petarc-eiq Looks like CI is still getting error running pipeline tests.
```
one or more errors found in document: [0] parsing field value failed: field "threat.indicator.name"'s value "unit42.test.com" is not one of the expected values (5.2.75.227, 2a02:cf40:add:4002:91f2:a9b2:e09a:6fc6, https://example.com/some/path, example.com, 373d34874d7bc89fd4cefa6272ee80bf, b0e914d1bbe19433cc9df64ea1ca07fe77f7b150b511b786e46e007941a62bd7, email@example.com, HKLM\\SOFTWARE\\Microsoft\\Active, 13335, 00:00:5e:00:53:af, 8008) 
```
Can you make the fix as per the above review?